insights/parsers/systemd/unitfiles.py

```diff
@@ -69,15 +56,99 @@ def parse_content(self, content):
                 self.services[service] = enabled
                 self.parsed_lines[service] = line
                 self.service_list.append(service)
+            elif any(part in valid_states for part in parts):
```

Comment:
How much benefit is there to having a base class here?  It seems that there is very little shared code between the two child classes.  A shared combiner might be a better alternative depending upon how this information is utilized in any rules.